code/datums/elements/bane.dm

```diff
@@ -0,0 +1,55 @@
+/// Deals extra damage to mobs of a certain type or species.
+/datum/element/bane
+	element_flags = ELEMENT_DETACH|ELEMENT_BESPOKE
+	id_arg_index = 2
+	/// can be a mob or a species.
+	var/target_type
+	/// multiplier of the item strength for damage calculation.
+	var/damage_multiplier
```

Comment:
might want to mention that this is only for the EXTRA damage, so you need to use a value of 1 here if you want a bane item to do 2x damage, a value of 2 if you want it to do 3x damage, etc.